### ISSUE TITLE
Cable upgrade recipe is more expensive than non-upgrade recipe

### ISSUE THREAD

**Issue by natedogith1** [Technici4n/Powah#101]
#### Version: (make sure you are on the latest version before reporting):

- Minecraft: 1.18.2
- Forge: 40.2.10
- Powah: 3.0.8

#### Issue description:

The cable upgrade recipe uses 2 of the previous tier cables where the non-upgrade recipe uses 2 crystals but only produces half as many cables.  Since both use a capacitor, and the capacitor needs 4 crystals, the upgrade recipe uses more of every resource to produce an equivalent number of cables as the non-upgrade recipe.

**Comment by Spongman** [comment]
yeah, this doesn't make any sense. why would the upgrade recipe be _significantly_ worse value than the non-upgrade recipe? it seems backwards to me. there's absolutely no reason to upgrade your cables, or for this recipe to even exist - it's a complete waste. it's cheaper to just throw your old cables into lava.

![Image](https://github.com/user-attachments/assets/e6b71cdb-8e44-4b2d-a47a-68c9ecf5a829)

**Comment by Spongman** [comment]
here's the recipes from an old version of the mod:

![Image](https://github.com/user-attachments/assets/a0ebc1eb-ecf3-441d-bd4d-aebd7f339459)

notice that back then the upgrade recipes _were_ cheaper than the non-upgrade recipes.

however, the non-upgrade recipes have since been made cheaper making the upgrade recipes more expensive.
